strongbox-resources/strongbox-common-resources/pom.xml

```diff
@@ -25,14 +25,26 @@
         <developerConnection>scm:git:git://github.com/strongbox/strongbox.git</developerConnection>
     </scm>
 
-    <build>
-        <plugins>
-            <plugin>
-                <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-source-plugin</artifactId>
-            </plugin>
-        </plugins>
-    </build>
+	<build>
```

Comment:
Cosmetic: Coul you please use the same indentation that we do? We use four spaces, instead of tabs. Thanks! :)

Reply:
Done

Reply:
Thanks! :)